data-raw/cases_deaths_subset.R

```diff
@@ -1,3 +1,7 @@
+library(dplyr)
+library(epidatr)
+library(epiprocess)
+
```

Comment:
Need to update this and others with the correct `epidatr` call. Is it `pub_covidcast()`?